### ISSUE TITLE
TEST to PROD - world-team-mapping

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#351]
@jamesmrollins 

@TomRust @staceypark and I just observed this, when I logged in to my individual world. I'm not John! 😄 

Stacey showed Tom and I hope to track it down with the team-world-mapping file and Epicenter. However, we want to double-check that this is just a human error issue.

Stacey fixed the alignment of the individual world mapping in PROD before the recent push from TEST to PROD, but we see it just got out of sync as up this morning. So, we just want to make sure it is not an issue associated with moving from TEST to PROD.

![image](https://user-images.githubusercontent.com/7454688/57041273-ae3fae00-6c16-11e9-85f9-05561595a09f.png)

**Comment by jamesmrollins** [comment]
@lzim I am not sure what you are saying.  

Are you saying that:
1. You observed the team name misalignment with your login - and
2. Stacey went into the world-team-mapping.xlsx file and discovered the source of the problem?

or

3.  The world-team-mapping file is correct and the sim is not reading correctly?

Please clarify.  If 1 or 2, then indeed, the misalignment could be a matter of moving an outdated Test file into PROD last night, or, I messed it up somehow this morning when I tested the MM/CC new world Learning Mode issue.

**Comment by lzim** [comment]
@jamesmrollins yes to 1. And 2. 
But, I’m addition she fixed the misalignment once before. Then the error returned, after a push to PROD.

- it just a human mistake - understand.
- just making sure not something happening automatically when going from TEST to PROD

@staceypark can you help further clarify what you think is happening?

**Comment by staceypark** [comment]
@jamesmrollins @lzim 
James and I talked last night and made several decisions.
We confirmed that my theory on TEST to PROD file transfer is true, meaning 
1. the error itself is a human error
2. It was not caught in TEST before we went to PROD
3. Error transferred from TEST to PROD
4. Fixed in PROD, but now out of sync with TEST (which shouldn't really be happening in theory, since all of this should have been caught in TEST)
5. When Dev pushes TEST to PROD, the out of sync TEST files because update into PROD

However, James and I decided that we would keep the world mapping xlsx separate between TEST to PROD, since there are different types of groups needed between the two.

tldr; continue operating as before

**Comment by jamesmrollins** [comment]
@staceypark @lzim We also discussed that a major house cleaning is in order.  The proliferation of Groups that we don't use anymore is also a part of the problem.

**Comment by lzim** [comment]
@staceypark @jamesrollins 

At this point for housekeeping: I believe we only need R21 team/individual worlds and our TAS BHIP team/individual worlds.

Let’s clean this up before we add Epicenter to the MTL Facilitate training for the TAS. And, let’s use it as part of our training on all the Epicenter functionalities we all need to know how to do without Stacey & James help.

FYI re R21 sites: @tomrust @dlounsbu

FYI: dlkibbe and @branscombj

**Comment by staceypark** [comment]
@lzim what about the mtl facilitate pilot world from chicago? where should I put the text from that somewhere before we nix it, if at all?

r21 sites: @TomRust @dlounsbu I won't be deleting their original large team group until we finalize the migration and fleshing out of all the experiments. I believe our last step here is for @dlounsbu to finish up Team 5. Let me know if you need anything from me to get that done. 

@lzim I'll also be backing up exports from that original large world into their team folders in teampsd before deletion *just in case*

**Comment by lzim** [comment]
@staceypark You're right we do need very good records of:

1. R21 teams
2. MTL Facilitate Pilot w/TAS
3. Our current training world (ind & team) for West Lafayette

@jamesmrollins I'm not sure I"m tracking how migration works, but I believe we may have two needs:

1. Some worlds just need to be cleaned/deleted, etc.
2. Some worlds need to be subsetted and archived somewhere. For example, #2 above MTL Facilitate Pilot w/TAS likely fits this. 

@staceypark I definitely want to keep very close track of #2 so that we can describe exactly what we did in that training. NO RISK of LOSS, whatever we do.

**Comment by staceypark** [comment]
@jamesmrollins @lzim 
I think in terms of "archiving" data, we just need to export all of the experiments run since that should hold all of the experimental values changed and qhfd text entered and we can back those up in TeamPSD. Am I missing anything?

**Comment by lzim** [comment]
@staceypark @jamesmrollins @tomrust @saveth

Are you suggesting, Stacey, that the tt3 report (the complete report of all they did), should give us everything we need?

**Comment by saveth** [comment]
Closing this since @staceypark and @jamesmrollins  resolved this issue. When sim versions move from TEST to PROD, the world team mapping data is not also transferred over. The world team mapping files for TEST and PROD will remain divorced of each other to prevent work from being erased.
